test/calculate-svg-dimensions.test.js

```diff
@@ -1,66 +1,76 @@
 'use strict';
 
-const assert = require('assert').strict;
-const { readFileSync } = require('fs');
+const { readFile } = require('fs').promises;
 const path = require('path');
 const calculateSvgDimensions = require('../lib/svg-sprite/utils/calculate-svg-dimensions.js');
 
-const { paths } = require('./helpers/constants.js');
```

Comment:
Makes me wonder if we could/should just remove helpers/constants.js later?

Reply:
We'll see. Maybe we can reduce them, but removing all of them seems impossible for now.